server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/flowtask/DataArchivePreprocessor.java

```diff
@@ -104,6 +114,13 @@ public void process(CreateFlowInstanceReq req) {
             if (parameters.getOperationType() == OperationType.UPDATE) {
                 parameters.setDescription(req.getDescription());
                 ScheduleEntity scheduleEntity = scheduleService.nullSafeGetById(parameters.getTaskId());
+                List<ScheduleTaskEntity> runningTasks = scheduleTaskService.listTaskByJobNameAndStatus(
+                        scheduleEntity.getId().toString(), Arrays.asList(TaskStatus.RUNNING, TaskStatus.PREPARING));
+                if (scheduleEntity.getStatus() != ScheduleStatus.PAUSE || !runningTasks.isEmpty()) {
```

Comment:
What's the relation between this condition `scheduleEntity.getStatus() != ScheduleStatus.PAUSE || !runningTasks.isEmpty()` and your log 'Resume schedule not allowed ...'

Reply:
In order to ensure scheduling security, editing is only allowed when the plan is paused and there are no subtasks executing.